src/clincoded/static/components/curation_central.js

```diff
@@ -283,14 +283,15 @@ var PmidSelectionList = createReactClass({
                                         <PmidSummary article={annotation.article} />
                                     </div>
                                     <div className="pmid-selection-list-pmid"><a href={external_url_map['PubMed'] + annotation.article.pmid} target="_blank">PMID: {annotation.article.pmid}</a></div>
+                                    { annotation.article.doi ? <div className="pmid-selection-list-pmid"><a href={external_url_map['PubMed'] + annotation.article.pmid} target="_blank">DOI: {annotation.article.doi}</a></div> : null}
```

Comment:
Is it possible for `annotation.article` to be undefined? Accessing `annotation.article.doi` or `annotation.article.pmid` could throw an error if `annotation.article` is undefined.

Reply:
Hey Howard! Apologies for it taking a while to come back to this. I have not been able to recreate the issue with an undefined article. Wanted to ask which steps you took when testing to recreate it and figure it out! Thanks